plugins/inputs/nfsclient/nfsclient.go

```diff
@@ -14,40 +14,42 @@ import (
 )
 
 type NFSClient struct {
-	Fullstat          bool
-	IncludeMounts     []string
-	ExcludeMounts     []string
-	IncludeOperations []string
-	ExcludeOperations []string
-	Log               telegraf.Logger
+	Fullstat          bool     `toml:"fullstat"`
+	IncludeMounts     []string `toml:"include_mounts"`
+	ExcludeMounts     []string `toml:"exclude_mounts"`
+	IncludeOperations []string `toml:"include_operations"`
+	ExcludeOperations []string `toml:"exclude_operations"`
+	nfs3Ops           map[string]bool
+	nfs4Ops           map[string]bool
```

Comment:
Just cosmetics, but it would be nice to move these two options below `Log` to keep all externally set fields together...

Reply:
Please mark as resolved.